implementation/map-to-scatter-gather/app.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+import os
+
+import aws_cdk as cdk
+
+from scatter_gather.sfn_map_component import SFNScatterGatherStack
+from scatter_gather.refactored_component import ScatterGatherStack
+
+
+app = cdk.App()
+SFNScatterGatherStack(app, "ScatterGatherStack",
+                      env=cdk.Environment(
+                            account=os.environ["CDK_DEFAULT_ACCOUNT"],
+                            region=os.environ["CDK_DEFAULT_REGION"]))
+
+ScatterGatherStack(app, "RefactoredScatterGather",
```

Comment:
Minor: One stack ends in `Stack` whereas the other one doesn't. Better to be consistent